src/bindings/python/fluxacct/accounting/util.py

```diff
@@ -54,3 +54,15 @@ def parse_timestamp(timestamp):
     except ValueError:
         # just return as a seconds-since-epoch timestamp
         return timestamp
+
+
+def format_value(val):
+    """
+    Replace a max value in the database with a string.
+
+    Args:
+        val: the value being evaluated
+    """
+    if isinstance(val, int) and val == 2147483647:
+        return "MAX"
```

Comment:
Could this number be saved as a constant somewhere, like `INTEGER_MAX`? Also is the `isinstance` check accomplishing anything here?

Reply:
Ah, this is a great suggestion. A constant would probably be cleaner to read here and elsewhere. Thanks for this.

> is the isinstance check accomplishing anything here?

🤦 I don't believe it is; I could probably just remove it. Thanks!